home.html

```diff
@@ -77,5 +77,152 @@ <h2 class="profession-title">I'm a Software Developer</h2>
         <img class="m-icon" src="/assets/images/icon-medium.png" alt="m-icon" />
       </div>
     </section>
+    <section class="work-section">
+      <div class="recent-work">
+        <h2 class="work-heading">My Recent Works</h2>
+        <hr />
+        <img
+          class="image-placeholder"
+          src="./assets/images/img-placeholder.jpg"
+          alt="placeholder image"
+        />
+        <h3 class="sub-heading">Multi-Post Stories</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <div class="button-group-1">
+          <a class="btn btn-transparent" href="#">CSS</a>
+          <a class="btn btn-transparent" href="#">HTML</a>
+          <a class="btn btn-transparent" href="#">Bootstrap</a>
+          <a class="btn btn-transparent" href="#">Ruby</a>
+        </div>
```

Comment:
I think the list of project technologies (e.g., "CSS", "HTML", "Bootstrap" and "Ruby") should be a list `ul` and each of them a `li`. Please add the `ul` tag instead of the `div` tag and replace the `a` tag with a `li` tag 😊 .